src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileStatusService.scala

```diff
@@ -11,7 +11,7 @@ import java.time.Instant
 import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
 
-class FileStatusService(fileStatusRepository: FileStatusRepository, uuidSource: UUIDSource)(implicit
+class FileStatusService(fileStatusRepository: FileStatusRepository)(implicit
```

Comment:
There are a couple of unused imports that should be removed:

- `import java.sql.Timestamp`
- `import java.time.Instant`